website/content/en/docs/tasks/operator.md

```diff
@@ -107,3 +106,48 @@ The `deploy peer` command has the following flags:
 - **tag:** This field determines the version of project images to pull. By default, it uses the `latest` version.
 
 All the flags are mapped to corresponding field in the ClusterLink custom resource.
+
+## Manual Deployment without CLI
+
+To deploy the ClusterLink project without using the CLI, follow the instructions below:
```

Comment:
nit: Please avoid using `project` etc. It should be enough to say `To deploy ClusterLink without...`